components/atoms/ActionButton.js

```diff
@@ -1,117 +1,56 @@
-  const disabledStyle =
-    'py-2 px-4 bg-gray-light text-gray-600 border border-gray-md'
+    'rounded-sm focus:ring-1 focus:ring-black focus:ring-offset-2 py-2 px-4 bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light'
 
-  //Activate Links with spacebar
-  useEffect(() => {
-    let link = document.getElementById(props.id)
-    if (link) {
-      link.addEventListener('keydown', (event) => {
-        if (event.key === 'Spacebar' || event.key === ' ') {
-          event.preventDefault()
-          link.click()
-        }
-      })
-    }
-  })
-
-  return props.href ? (
-    <Link href={props.href}>
+  return href ? (
+    <Link href={href}>
       <a
-        className={`${basicStyle}
-        ${
-          !props.secondary &&
-          !props.tertiary &&
-          !props.disabled &&
-          !props.custom
-            ? defaultStyle
-            : props.className
-        }
-        ${props.secondary && !props.disabled ? secondaryStyle : props.className}
-        ${props.tertiary && !props.disabled ? tertiaryStyle : props.className}
-        ${props.custom && !props.tertiary ? props.custom : ''}
-        ${props.disabled ? disabledStyle : props.className}`}
-        onClick={props.onClick}
-        id={props.id}
-        data-testid={props.dataTestId}
-        data-cy={props.dataCy || props.id}
-        data-cy-button={props.dataCyButton}
-        disabled={props.disabled}
+        className={`${!custom ? defaultStyle : className}`}
+        onClick={onClick}
         role="button"
         draggable="false"
-        lang={props.lang}
-        locale={props.langUrl}
+        lang={lang}
+        {...rest}
+        id={id}
       >
-        {props.icon && !props.iconEnd ? (
-          <span className={props.icon} data-testid={props.dataTestId} />
-        ) : undefined}
-        {props.text}
-        {props.children}
-        {props.icon && props.iconEnd ? (
-          <span className={props.icon} data-testid={props.dataTestId} />
-        ) : undefined}
+        {text}
+        {children}
       </a>
     </Link>
   ) : (
     <button
-      className={`${basicStyle}
-      ${
-        !props.secondary && !props.tertiary && !props.disabled && !props.custom
-          ? defaultStyle
-          : props.className
-      }
-      ${props.secondary && !props.disabled ? secondaryStyle : props.className}
-      ${props.tertiary && !props.disabled ? tertiaryStyle : props.className}
-      ${props.custom && !props.tertiary ? props.custom : ''}
-      ${props.disabled ? disabledStyle : props.className}`}
-      onClick={props.onClick}
-      type={props.type}
-      id={props.id}
-      data-testid={props.dataTestId}
-      data-cy={props.dataCy || props.id}
-      data-cy-button={props.dataCyButton}
-      disabled={props.disabled}
-      data-gc-analytics-submit={props.analyticsTracking ? 'submit' : undefined}
+      className={`${!custom ? defaultStyle : className}`}
```

Comment:
Instead of passing a boolean for custom can we do a check to see if className is defined?
Then instead of passing props custom=true and className=example class we can just pass className and it is inferred to overwrite the default class?